examples/passport-oauth2-login/src/models/access-token.model.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-oauth2-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+@model({settings: {strict: false}})
```

Comment:
Why are settings required in the @model decorator?

Reply:
This link https://loopback.io/doc/en/lb4/Model.html#using-the-juggler-bridge shows 2 examples. One where no settings are placed in @model decorator, and one with . If you could elaborate. thx.

Reply:
I am still looking at this

Reply:
I have removed the strict: false setting now